frontend/src/pages/OneMovie.jsx

```diff
@@ -1,21 +1,16 @@
 import { React } from "react";
 import { useParams } from "react-router-dom";
-import CardCastingCarousel from "../components/Casting/CardCastingCarousel";
-import CardReviewCarousel from "../components/CardReviewCarousel";
-import MovieVideoCarousel from "../components/MovieVideoCarousel";
-import CardDetailSynopsis from "../components/CardDetailSynopsis";
+
 import Header from "../components/Header";
+import ButtonGroupFilmNavigation from "../components/ButtonGroupFilmNavigation";
 
 export default function OneMovie() {
   const { movieid } = useParams();
 
   return (
     <div>
       <Header />
-      <CardReviewCarousel movie={movieid} />
-      <CardCastingCarousel movie={movieid} />
-      <MovieVideoCarousel movie={movieid} />
-      <CardDetailSynopsis />
+      <ButtonGroupFilmNavigation movie={movieid} />
     </div>
```

Comment:
This PR seems to remove elements added during the PR that I just validated...

Reply:
euh je crois que c'est bon ^^